### DISCUSSION TITLE
Azure AD Auth Broken - 504

### DISCUSSION THREAD

**Discussion by mcaulifn** [netbox-community/netbox#13063] [Q&A] [answered]
I upgraded from 3.3.5 to 3.5.4. Azure AD is the auth. It has been working great till the upgrade. Now, NetBox is hanging on the GET to `/oauth/complete/azuread-oauth2/?code=`. The ELB is reporting a 504. I never see it in the uswgi logs as a request. Instead, the recv-q for the port just increases until I restart the service. Any suggestions on where to look?

**Comment by mcaulifn** [comment, marked as answer]
It turns out this was related to IPv6 being blocked. Disabling IPv6 or unblocking allowed the connection from NB to Azure.

I had to enable the root logger to find the connection attempt to Azure. It is a little concerning that that was the only time I could actually see that attempt.

**Reply by noc-cgesd** [reply]
This just happened to my instance as well. Disabling IPv6 resolved the issue
